packages/@ember/application/tests/dependency_injection_test.js

```diff
@@ -13,7 +13,9 @@ moduleFor(
     constructor() {
       super();
 
-      application = run(EmberApplication, 'create');
+      expectDeprecation(() => {
```

Comment:
This test is definitely using globals resolution mode, but it doesn't seem like it fundamentally needs to. We should probably make base class with a `setupRegistry` method then extend from that in two subclasses where one builds a global setup like the tests do currently and another uses the registry directly.